src/elements/button/button.svelte

```diff
@@ -65,17 +66,22 @@ const handleParentClick = (event: PointerEvent) => {
     aria-label={variant === 'icon' ? label : undefined}
     aria-disabled={isDisabled ? true : undefined}
     {title}
-    class={cx('active:scale-95 motion-safe:transition-transform', {
-      'inline-flex items-center justify-center gap-1.5 py-1.5 px-3 text-xs border':
+    class={cx({
+      'h-[30px] w-[30px]': variant === 'icon',
+      'px-4': !icon || variant === 'icon',
+      'pl-3 pr-4': icon && variant !== 'icon',
+      'inline-flex items-center justify-center gap-1.5 py-1 text-sm border':
         variant !== 'icon',
-      'bg-light border-light hover:bg-medium hover:border-medium':
+      'bg-light border-light hover:bg-medium hover:border-medium active:bg-gray-2':
         variant === 'primary',
-      'bg-gray-9 border-gray-9 text-white hover:bg-black hover:border-black':
+      'bg-gray-9 border-gray-9 text-white hover:bg-black hover:border-black active:bg-[#000]':
         variant === 'inverse-primary',
-      'bg-danger-dark text-white border-danger-dark hover:bg-[#aa2a2b]':
+      'border-transparent hover:bg-[rgba(0,0,0,0.04)] active:bg-[rgba(0,0,0,0.08)] active-border-[rgba(0,0,0,0.08)]':
+        variant === 'ghost' || variant === 'icon',
+      'bg-danger-dark text-white border-danger-dark hover:bg-[#aa2a2b] active:bg-[#9e2728]':
```

Comment:
Why the custom colors?

Reply:
These are colors that design wants for specific states in this button, but shouldn't be used elsewhere. For that reason it's best to keep them here and out of the theme.